specification/2.0/README.md

```diff
@@ -624,7 +624,32 @@ When `byteStride` of referenced `bufferView` is not defined, it means that acces
 
 Each `accessor` must fit its `bufferView`, i.e., `accessor.byteOffset + STRIDE * (accessor.count - 1) + SIZE_OF_ELEMENT` must be less than or equal to `bufferView.length`.
 
-For performance and compatibility reasons, vertex attributes must be aligned to 4-byte boundaries inside `bufferView` (i.e., `accessor.byteOffset` and `bufferView.byteStride` must be multiples of 4). Accessors of matrix type have data stored in column-major order; start of each column must be aligned to 4-byte boundaries.
+For performance and compatibility reasons, vertex attributes must be aligned to 4-byte boundaries inside `bufferView` (i.e., `accessor.byteOffset` and `bufferView.byteStride` must be multiples of 4). 
+
+Accessors of matrix type have data stored in column-major order; start of each column must be aligned to 4-byte boundaries. To achieve this, three `type`/`componentType` combinations require special layout:
+
+**MAT2, 1-byte components**
+```
+| 00| 01| 02| 03| 04| 05| 06| 07| 
+|===|===|===|===|===|===|===|===|
+|m00|m10|---|---|m01|m11|---|---|
+```
+
+**MAT3, 1-byte components**
+```
+| 00| 01| 02| 03| 04| 05| 06| 07| 08| 09| 0A| 0B|
+|===|===|===|===|===|===|===|===|===|===|===|===|
+|m00|m10|m20|---|m01|m11|m21|---|m02|m12|m22|---|
+```
+
+**MAT3, 2-byte components**
+```
+| 00| 01| 02| 03| 04| 05| 06| 07| 08| 09| 0A| 0B| 0C| 0D| 0E| 0F| 10| 11| 12| 13| 14| 15| 16| 17|
+|===|===|===|===|===|===|===|===|===|===|===|===|===|===|===|===|===|===|===|===|===|===|===|===|
+|m00|m00|m10|m10|m20|m20|---|---|m01|m01|m11|m11|m21|m21|---|---|m02|m02|m12|m12|m22|m22|---|---|
+```
+
+Alignment requirements apply only to start of each column, so trailing zeros could be omitted if there's no further data. 
```

Comment:
> trailing zeros

This is the first mention of padding with zeros. Is there some text somewhere that indicates they have to be zero padded?